tests/Backend/Snowflake/TimeTravelTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Keboola\Test\Backend\Snowflake;
+
+use Keboola\Test\StorageApiTestCase;
+use Keboola\Csv\CsvFile;
+
+class TimeTravelTest extends StorageApiTestCase
+{
+    private $destinationBucketId;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->_initEmptyTestBuckets();
+        $bucketData = array(
+            'name' => 'timetravel-test',
+            'stage' => 'in',
+            'description' => 'time travel test bucket',
+        );
+        $this->destinationBucketId = $this->_client->createBucket(
```

Comment:
Why are you creating a special bucket here?  You can just use `$this->getTestBucketId(self::STAGE_OUT)` for example. Than you can remove that `tearDown`, we are not using it in sapi client test because we it to be idempotent.